server/service/src/sync/test/integration/remote/mod.rs

```diff
@@ -1,3 +1,4 @@
+// pub(crate) mod activity_log;
```

Comment:
should this and the next file have `TODO`s in them?

Reply:
Couldn't reply to below comment for some reason. A new table has been created for the logging and the types are mapped from line 48